pinot-common/src/main/java/com/linkedin/pinot/common/metadata/segment/OfflineSegmentZKMetadata.java

```diff
@@ -32,6 +32,9 @@
   private String _downloadUrl = null;
   private long _pushTime = Long.MIN_VALUE;
   private long _refreshTime = Long.MIN_VALUE;
+  private String _segmentPushStatus = NONE;
```

Comment:
your comments say it will be populated only when segment is committing. Are you populating it with the string "NONE" even otherwise?

Reply:
from my understanding, in zk, it will always be populated with something, whether it is the string "null" or something else. By that logic, I thought it would be better to always populate it with NONE to make it less confusing, which I was planning on doing inside upload segment though I could also do it here. I don't know what the convention we want to follow for "empty" fields in zk is; for example, if you look at the field schema_name, I've seen 
1. empty string
2. the string "null" (which I feel is very confusing when doing comparisons)
3. the string NO_SCHEMA

Do you have a suggestion about the convention that I should be following?